### ISSUE TITLE
Is there an official icon for rpiplay?

### ISSUE THREAD

**Issue by Botspot** [FD-/RPiPlay#237]
I'm interested in adding rpiplay to the [Pi-Apps RPi app store](https://github.com/Botspot/pi-apps).

When displayed in the app list, rpiplay will need an icon to go with it. Is there an official (or unofficial) icon that can be associated with rpiplay?

Thanks in advance.

**Comment by CleanMachine1** [comment]
@FD- ?

**Comment by FD-** [comment]
We don’t have an official icon yet I’m afraid. Any suggestions?

**Comment by CleanMachine1** [comment]
First of all, I hope you are OK with Botspot and me adding RPi-Play.

I suggested the official logo provided by Apple, would you agree with that choice

**Comment by FD-** [comment]
No, please do NOT use the official AirPlay icon since I’m sure you do not have Apple’s permission to do so.

**Comment by CleanMachine1** [comment]
Ok thanks for your help.
